site/css/style.css

```diff
@@ -427,8 +466,35 @@ fieldset {
     margin-bottom: 1em;
 }
 
-.page-controls button {
-    padding: 0.25em;
+
+.endpoint-area .input-field {
+    display: flex;
+    flex-direction: column;
+}
+
+.endpoint-area label {
+    margin-bottom: 0.25em;
+}
+
+.endpoint-area input {
+    padding: 0.5em 1em;
```

Comment:
I would break out the padding into the specific parts.
It makes it easier to read and debug.
For padding, margin, and border, you usually want to go with either all in one or specific components.

(I personally would just recommend 0.5em for all (so just: `padding: 0.5em`),
but you can choose this.)